features/shop/apply_correct_shipping_fee_on_order.feature

```diff
@@ -5,11 +5,13 @@ Feature: Apply correct shipping fee on order
     I want to have shipping fee applied based on chosen shipping method
 
     Background:
-        Given the store is operating on a single "France" channel
+        Given the store is operating on a single channel
+        And store ships to "Australia", "France" and "United States"
```

Comment:
Aggreed :+1: however, there are many steps where it should be changed, so I think we should do this in separate PR, wdyt?